tests/ci/ci_runner.py

```diff
@@ -380,15 +382,15 @@ def _install_clickhouse(self, debs_path):
         os.chmod(CLICKHOUSE_ODBC_BRIDGE_BINARY_PATH, 0o777)
         os.chmod(CLICKHOUSE_LIBRARY_BRIDGE_BINARY_PATH, 0o777)
         shutil.copy(
-            CLICKHOUSE_BINARY_PATH, os.getenv("CLICKHOUSE_TESTS_SERVER_BIN_PATH")
+            CLICKHOUSE_BINARY_PATH, os.getenv("CLICKHOUSE_TESTS_SERVER_BIN_PATH")  # type: ignore
```

Comment:
should be `str`, why mypy complains?

Reply:
might be None

Reply:
If it's `None`, the `shutil.copy` must fail. So, it is worth setting a decent default or failing explicitly.